Add ability to customize order of output for edge tpu detector models

alec-groff opened the pull request:
Add order to config.model to allow the user the ability to specify the order of boxes, scores, class_ids, and count for their edge tpu detector output. The default value is the current setting.

alec-groff committed:
Update detector_config.py

alec-groff committed:
Update edgetpu_tfl.py

alec-groff committed:
Update edgetpu_tfl.py

NickM-27 commented on frigate/detectors/plugins/edgetpu_tfl.py:
```diff
@@ -44,6 +44,7 @@ def __init__(self, detector_config: EdgeTpuDetectorConfig):
                 model_path=detector_config.model.path,
                 experimental_delegates=[edge_tpu_delegate],
             )
+            self.order = detector_config.model.order
```
I don't think there is a reason to set `self.order` every time, just set it once in init

NickM-27 commented on frigate/detectors/detector_config.py:
```diff
@@ -51,6 +51,9 @@ class ModelConfig(BaseModel):
     model_type: ModelTypeEnum = Field(
         default=ModelTypeEnum.ssd, title="Object Detection Model Type"
     )
+    order: list[int] = Field(
```
this config would apply to every detector so it needs to be used for every detector or perhaps the default value should be `None` and the field changes to be called `custom_model_output_order` or something

NickM-27 replied:
I think the name should be more descriptive like `model_output_tensor_order` or something

alec-groff committed:
modified:   frigate/detectors/detector_config.py
	modified:   frigate/detectors/plugins/cpu_tfl.py
	modified:   frigate/detectors/plugins/edgetpu_tfl.py

bazylhorsey commented:
Any update on this?
